lib/bootloader_pvm.pm

```diff
@@ -117,10 +117,15 @@ sub prepare_pvm_installation {
 
     # Delete partition table before starting installation
     select_console('install-shell');
-    foreach my $disk (split(',', get_var('DISK_DEVICES', 'sda'))) {
```

Comment:
As we get rid of this variable, let's remove it from `variables.md` too.

Reply:
yep, but also found better solution =)